app/controllers/application_controller.rb

```diff
@@ -20,6 +22,10 @@ def error_messages(obj)
     obj.errors.full_messages.join(", ")
   end
 
+  def set_search
+    @q=User.search(params[:q])
+  end
+
```

Comment:
This should live in the `UsersController` since we may search for other objects in the future (ie. outreach events, cohorts, etc)